java/system-test/system-test-dsl/src/test/java/sleeper/systemtest/dsl/testutil/InMemorySystemTestDrivers.java

```diff
@@ -182,4 +184,9 @@ public TableMetricsDriver tableMetrics(SystemTestContext context) {
     public InMemoryDataStore data() {
         return data;
     }
+
+    @Override
+    public SnapshotsDriver snapshots(SystemTestContext context) {
+        return new InMemorySnapshotsDriver();
```

Comment:
It's difficult to tell whether we want this to be used in any in-memory DSL tests, so that we could copy the exact code from the in-memory test to the real system test, or if we can just make this throw UnsupportedOperationException in SystemTestDriversBase.

It might be nice to have a unit test where we hold a boolean in the driver and assert on that.